specs/eventing/control-plane.md

```diff
@@ -0,0 +1,985 @@
+
+1. If the delivery attempt is successful (either the original request or a
+   retry) and an event is returned in the reply, the reply event will be
+   delivered to the `status.replyUri` destination (for Subscriptions) or added
+   to the Broker for processing (for Triggers). If `status.replyUri` is not
+   present in the Subscription, the reply event is be dropped.
+
+   1. If delivery of the reply event fails with a retryable error, the delivery
+      of the reply event to the reply destination SHOULD be retried up to
+      `retry` times, following the `backoffPolicy` and `backoffDelay` parameters
+      if specified.
+
+1. If all retries are exhausted for either the original delivery or the retry,
+   or if a non-retryable error is received, the event should be delivered to the
+   `deadLetterSink` in the delivery options. If no `deadLetterSink` is
+   specified, the event is dropped.
+
+   The implementation MAY set additional attributes on the event or wrap the
+   failed event in a "failed delivery" event; this behavior is not (currently)
+   standardized.
+
+   If delivery of the reply event fails with a retryable error, the delivery to
+   the `deadLetterSink` SHOULD be retried up to `retry` times, following the
+   `backoffPolicy` and `backoffDelay` parameters if specified. Alternatively,
+   implementations MAY use an equivalent internal mechanism for delivery (for
+   example, if the `ref` form of `deadLetterSink` points to a compatible
+   implementation).
+
+## Detailed Resources
+
+### Broker v1
+
+#### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource.
+
+#### Spec:
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>config</code></td>
+    <td><a href="#kreference">KReference</a><br/>(Optional)</td>
+    <td>A reference to an object which describes the configuration options for the Broker (for example, a ConfigMap).</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>delivery</code></td>
+    <td><a href="#deliveryspec">DeliverySpec</a><br/>(Optional)</td>
+    <td>A default delivery options for Triggers which do not specify more-specific options.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+#### Status
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>conditions</code></td>
+    <td><a href="#error-signalling">See Error Signalling</a></td>
+    <td>Used for signalling errors, see link. Conditions of types Ready MUST be present.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>observedGeneration</code></td>
```

Comment:
curious.... why do we mention this? It feels like an internal bit of data and not something that we need to get interop on. Or do we expect people to be able to act upon this value?

Reply:
That sounds like something we need to state more explicitly someplace since outside of this row I don't think we talk about that - or perhaps I missed it. But either way, I still think we should drop it and make life easy on people and ask them to only check one field, not two. You're basically implying that all of the text we have in other spots that talk about sending events based on Ready==true is insufficient because they need to also look at the observedGeneration field.